Week2/todo_app/app.js

```diff
@@ -0,0 +1,194 @@
+// This is the connector (also known as driver)
```

Comment:
I don't see how this one is necessary, since there's already the `program.js`.

Reply:
This is a separate file 👍